docs/configure_digital_credentials.md

```diff
@@ -0,0 +1,24 @@
+### Digital Credentials Configuration
+
+Digital credentials requires an instance of the []`sign-and-verify`]() service to be running.
+
+#### Settings
+
+
+| Setting | Value | Notes |
+|---|---|---|
+| `DIGITAL_CREDENTIALS_DEEP_LINK_URL` | `dcwallet://deep_link` | The deep link url to the digital credentials wallet app. This will typically involve a custom url scheme. |
+| `DIGITAL_CREDENTIALS_ISSUER_ID` | - | The digital credentials issuer id to be included in the credential template. This value is determined by the digital credentials team. |
+| `DIGITAL_CREDENTIALS_VERIFICATION_METHOD` | - | The digital credentials verification method to be included in the credential template. This value is determined by the digital credentials team. |
+| `MITOL_DIGITAL_CREDENTIALS_HMAC_SECRET` | - | Any random string, MUST match the corresponding HMAC secret `sign-and-verify` is deployed with. |
+| `MITOL_DIGITAL_CREDENTIALS_VERIFY_SERVICE_BASE_URL` | `http://sign-and-verify.example.com:5678/` | The addressable base url for the `sign-and-verify` service. |
```

Comment:
Unless there will be a ci/rc service we can use, maybe add documentation on how to get the local service running? `git clone digitalcredentials/sign-and-verify` , `npm run build & npm run start`, make accessible from inside docker, etc